hack/generate-manifest.sh

```diff
@@ -96,36 +100,32 @@ elif ! $KUSTOMIZE version > /dev/null 2>&1; then
     exit 1
 fi
 
-KUSTOMIZATION_DIR=$THIS_DIR/../build/yamls
-
-TMP_DIR=$(mktemp -d $KUSTOMIZATION_DIR/overlays.XXXXXXXX)
-
-pushd $TMP_DIR > /dev/null
-
-BASE=../../base
-
-mkdir $MODE && cd $MODE
-touch kustomization.yml
-$KUSTOMIZE edit add base $BASE
-# ../../patches/$MODE may be empty so we use find and not simply cp
-find ../../patches/$MODE -name \*.yml -exec cp {} . \;
-
-if [ "$MODE" == "dev" ]; then
-    $KUSTOMIZE edit set image clickhouse-monitor=projects.registry.vmware.com/antrea/theia-clickhouse-monitor:latest
-    $KUSTOMIZE edit add patch --path imagePullPolicy.yml --group clickhouse.altinity.com --version v1 --kind ClickHouseInstallation --name clickhouse
-fi
+HELM_TMP_DIR=$(mktemp -d $THIS_DIR/../build/yamls/chart-values.XXXXXXXX)
 
+EXTRA_VALUES=""
 if [ "$MODE" == "release" ]; then
-    $KUSTOMIZE edit set image clickhouse-monitor=$IMG_NAME:$IMG_TAG
+    EXTRA_VALUES="--set clickhouse.monitorImage.repository=$IMG_NAME,clickhouse.monitorImage.tag=$IMG_TAG"
 fi
 
+THEIA_CHART=$THIS_DIR/../build/charts/theia
+KUSTOMIZATION_DIR=$THIS_DIR/../build/yamls
+# intermediate manifest
+MANIFEST=$KUSTOMIZATION_DIR/base/manifest.yaml
+# Suppress potential Helm warnings about invalid permissions for Kubeconfig file
+# by throwing away related warnings.
+$HELM template \
```

Comment:
I would like to understand if we can completely drop kustomize by using helm --create-namespace with https://helm.sh/docs/faq/changes_since_helm2/#automatically-creating-namespaces

Would there be any issue? for instance, would helm fail if the namespace already exist?

Reply:
We support using `--create-namespace` when deploying by `helm install`. But we still keep the script for generating a standard `flow-visibility.yaml`. Kustomize is necessary here as `helm template` does not support `--create-namespace`.